### PR TITLE
OTS-344

### PR DESCRIPTION
**Pull request by adrice727**
Update the `batchSignal` method to ensure that we are not exceeding the maximum signal size set by OT.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/annotation-acc-pack/pull/89" target="_blank">ReviewNinja

File: js/opentok.js-annotations/dist/opentok-annotation.js [hunk 1]
```diff
@@ -1109,33 +1107,34 @@
...
+       * can be sent in the signal is 8,192.  Currently, the largest updates are
+       * 995 characters, meaning that the limit for the number of updates per
+       * signal should be 8, even taking into account the additional characters
+       * required to convert the entire array of updates as opposed to each one
+       * individually.  However, OT is throwing a 413 error once the size exceeds
+       * 7,900 characters. So, 7 is the magic number for the time being.
+       */
+      var dataChunk;
+      var start = 0;
+      var updatesPerSignal = 7;
```
Comment (Lucashuang0802) [inline js/opentok.js-annotations/dist/opentok-annotation.js]: Is it better we use `const` here?

File: js/opentok.js-annotations/dist/opentok-annotation.js [hunk 2]
```diff
@@ -1109,33 +1107,34 @@
...
+      while (start < data.length) {
+        dataChunk = data.slice(start, start + updatesPerSignal);
         updateType(dataChunk);
+        start += updatesPerSignal;
         var signal = {
           type: type,
           data: JSON.stringify(dataChunk)
         };
+        dataChunk.forEach(c => console.log('LLLL?', JSON.stringify(c).length));
```
Comment (Lucashuang0802) [inline js/opentok.js-annotations/dist/opentok-annotation.js]: Isn't it better to put this line after `self.session.signal(signal, signalError);` and take advantage of `signalError` for consideration like so?
```
if (signalError) {
  console.log(signalError);
}
else {
  dataChunk.forEach(c => console.log('LLLL?', JSON.stringify(c).length));
}
```

File: js/opentok.js-annotations/dist/opentok-annotation.js [hunk 3]
```diff
@@ -1109,33 +1107,34 @@
...
+       * individually.  However, OT is throwing a 413 error once the size exceeds
+       * 7,900 characters. So, 7 is the magic number for the time being.
+       */
+      var dataChunk;
+      var start = 0;
+      var updatesPerSignal = 7;
+      while (start < data.length) {
+        dataChunk = data.slice(start, start + updatesPerSignal);
         updateType(dataChunk);
+        start += updatesPerSignal;
```
Comment (Lucashuang0802) [inline js/opentok.js-annotations/dist/opentok-annotation.js]: 👍

File: js/opentok.js-annotations/dist/opentok-annotation.js (hunk 2 above)
Comment (adrice727) [inline js/opentok.js-annotations/dist/opentok-annotation.js]: I actually just forgot to remove that line.  It's been cleaned up in the latest commit.

**Comment by adrice727** [comment]
LGTM

**Comment by Lucashuang0802** [comment]
👍

**Pull request merged**